### PR TITLE
Update _index.md

### PR DESCRIPTION
**Pull request by valeriahhdez**
Add a section on how to solve spell-check failures. Replace heading verbs with the infinitive form to make them parallel.

### Describe your changes
This PR replaces heading verbs with infinitive forms to create parallel structures.  
It also adds a section that describes what to do in case a PR's spell-check fails. 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/es/.wordlist.txt [hunk 1]
```diff
@@ -39,11 +40,14 @@ category
...
 CC
 CD
+changes
 Chaos
+check
 Chris
 CI
 cifrada
 CLA
+click
```
Comment (electrocucaracha) [inline content/es/.wordlist.txt:50]:
```suggestion
```

File: content/es/contribute/_index.md [hunk 2]
```diff
@@ -169,11 +174,29 @@ Para cambiar un término directamente, vaya a "Editar esta página".
...
+
+- el PR contiene error(es) ortográfico(s),
+- el PR contiene palabras que no están registradas en la lista de palabras en español.
+
+Para poder añadir nuevas palabras a la lista en español, siga los siguientes pasos:
+
+1. Seleccione la carpeta "content" en su PR. 
+2. Vaya a la carpeta "es" y localice el archivo _".wordlist.txt".
+3. Seleccione la opción "Edit this file" y añada las palabras faltantes. 
+4. Al terminar, describa los cambios en el mensaje del commit y dé click en "Sign off and propose changes".
```
Comment (electrocucaracha) [inline content/es/contribute/_index.md:197]: La versión de `click` en español según la [RAE](https://dle.rae.es/clic) es `clic`
```suggestion
4. Al terminar, describa los cambios en el mensaje del commit y dé clic en "Sign off and propose changes". 
```

File: content/es/contribute/_index.md [hunk 3]
```diff
@@ -169,11 +174,29 @@ Para cambiar un término directamente, vaya a "Editar esta página".
...
+- el PR contiene palabras que no están registradas en la lista de palabras en español.
+
+Para poder añadir nuevas palabras a la lista en español, siga los siguientes pasos:
+
+1. Seleccione la carpeta "content" en su PR. 
+2. Vaya a la carpeta "es" y localice el archivo _".wordlist.txt".
+3. Seleccione la opción "Edit this file" y añada las palabras faltantes. 
+4. Al terminar, describa los cambios en el mensaje del commit y dé click en "Sign off and propose changes". 
+
+Nótese que las palabras están ordenadas alfabéticamente. Le pedimos encarecidamente cuide este orden.
```
Comment (electrocucaracha) [inline content/es/contribute/_index.md:199]:
```suggestion
Nótese que las palabras están en orden alfabético, le pedimos encarecidamente mantenga este orden. 
```

File: content/es/contribute/_index.md [hunk 4]
```diff
@@ -11,14 +11,14 @@ El contenido del Glosario Cloud Native se almacena en [este repositorio de GitHu
 donde encontrará una lista de [issues](https://github.com/cncf/glossary/issues), [PRs](https://github.com/cncf/glossary/pulls) y
 [discusiones](https://github.com/cncf/glossary/discussions) sobre el glosario.
 
-Existen tres formas en las que puedes contribuir:
+Existen cuatro formas en las que puede contribuir:
 
 1) [Trabajar en un issue existente](#work-on-an-existing-issue)
 2) [Proponer nuevos términos](#propose-new-terms)
 3) [Actualizar los existentes](#update-an-existing-term)
```
Comment (electrocucaracha) [inline content/es/contribute/_index.md:18]:
```suggestion
3) [Actualizar un término existente](#update-an-existing-term)
```

File: content/es/contribute/_index.md [hunk 5]
```diff
@@ -169,11 +174,29 @@ Para cambiar un término directamente, vaya a "Editar esta página".
 Esto abrirá la página del término en GitHub. Realice sus cambios y envíe un PR.
 Consulte "Envío de un nuevo término" (más arriba en este documento) para obtener una descripción mas detallada (salte a la sección que habla acerca de markdown).
 
-## Ayuda a traducir el glosario
+## Ayudar a traducir el glosario
```
Comment (electrocucaracha) [inline content/es/contribute/_index.md:177]:
```suggestion
## Ayudar a traducir el glosario {#help-translate-the-glossary}
```

File: content/es/contribute/_index.md [hunk 6]
```diff
@@ -64,12 +64,17 @@ Tenga en cuenta que debe cumplir los términos de la [definición nativa para la
...
-notificar a los mantenedores en Slack (de lo contrario, podemos perderlo).
-Y asegúrese de leer la [Guía de estilo](/es/style-guide/) antes de comenzar. ¡Gracias!
+**Si usted es un profesional de GitHub**, puede dar sólo un vistazo para asegurarse de lo siguiente:
 
-### Creando un issue {#creating-an-issue}
+1. usar nuestras plantillas de issues,
+2. encontrar la plantilla del archivo, 
+3. conocer las convenciones de nomenclatura, 
+4. notificar a los mantenedores en Slack (de lo contrario, podemos perderlo), 
+5. resolver fallas en la {revisión de ortografía}(#spell-check).
```
Comment (electrocucaracha) [inline content/es/contribute/_index.md:73]:
```suggestion
5. resolver fallas en la [revisión de ortografía](#spell-check).
```

File: content/es/.wordlist.txt [hunk 7]
```diff
@@ -187,10 +193,12 @@ needed
 Netflix
 networking
 new
+nótese
 Nicolas
 Not
```
Comment (electrocucaracha) [inline content/es/.wordlist.txt:198]:
```suggestion
Nicolas
Not
nótese
```

**Review by electrocucaracha — CHANGES_REQUESTED** [review]

File: content/es/contribute/_index.md [hunk 8]
```diff
@@ -169,11 +174,29 @@ Para cambiar un término directamente, vaya a "Editar esta página".
...
+
+La revisión de ortografía puede fallar por las siguientes razones:
+
+- el PR contiene error(es) ortográfico(s),
+- el PR contiene palabras que no están registradas en la lista de palabras en español.
+
+Para poder añadir nuevas palabras a la lista en español, siga los siguientes pasos:
+
+1. Seleccione la carpeta "content" en su PR. 
+2. Vaya a la carpeta "es" y localice el archivo _".wordlist.txt".
```
Comment (electrocucaracha) [inline content/es/contribute/_index.md:195]:
```suggestion
2. Vaya a la carpeta "es" y localice el archivo ".wordlist.txt".
```

File: content/es/contribute/_index.md [hunk 9]
```diff
@@ -11,14 +11,14 @@ El contenido del Glosario Cloud Native se almacena en [este repositorio de GitHu
 donde encontrará una lista de [issues](https://github.com/cncf/glossary/issues), [PRs](https://github.com/cncf/glossary/pulls) y
 [discusiones](https://github.com/cncf/glossary/discussions) sobre el glosario.
 
-Existen tres formas en las que puedes contribuir:
+Existen cuatro formas en las que puede contribuir:
```
Comment (electrocucaracha) [inline content/es/contribute/_index.md:14]:
```suggestion
Existen cuatro formas en las que se puede contribuir:
```

**Review by electrocucaracha — CHANGES_REQUESTED** [review]

File: content/es/contribute/_index.md [hunk 10]
```diff
@@ -169,11 +174,29 @@ Para cambiar un término directamente, vaya a "Editar esta página".
...
+
+1. Seleccione la carpeta "content" en su PR. 
+2. Vaya a la carpeta "es" y localice el archivo ".wordlist.txt".
+3. Seleccione la opción "Edit this file" y añada las palabras faltantes. 
+4. Al terminar, describa los cambios en el mensaje del commit y dé clic en "Sign off and propose changes". 
+
+Nótese que las palabras están en orden alfabético, le pedimos encarecidamente mantenga este orden. 
+La revisión ortográfica ignora el uso de mayúsculas y minúsculas. 
+
+
```
Comment (ramrodo) [inline content/es/contribute/_index.md:202]: Solo dejar una línea en blanco al final de cada archivo.

```suggestion
```

**Review by ramrodo — CHANGES_REQUESTED** [review]
¡Excelente contribución!

Solo un pequeño comentario.

**Review by electrocucaracha — APPROVED** [review]

**Review by krol3 — APPROVED** [review]
Excellent @valeriahhdez

**Pull request merged**